pkg/util/ballast.go

```diff
@@ -0,0 +1,23 @@
+package util
+
+// AllocateBallast allocates ballast of given size. to alter GC behaviour. See https://github.com/golang/go/issues/23044
+// Instead of allocating one big slice, we allocate many small ones to avoid keeping too much data in memory when Go runtime
+// decides that it needs to zero the slice.
+func AllocateBallast(ballastSize int) interface{} {
```

Comment:
[Nit] Isn't it clearer if the function declares the type it returns?
```suggestion
func AllocateBallast(ballastSize int) [][]byte {
```

Reply:
Idea is that caller doesn't care what this function returns, because it's never going to use it.

Reply:
Yes, but isn't it just beneficial to have a specific return type? It will also make tests easier to write, since they don't have to cast the return value?

Reply:
> Yes, but isn't it just beneficial to have a specific return type?

Only for the test. I don't think it's beneficial for anything else here. `interface{}` conveys that this is "something" that should not be used. `[][]byte` invites client to use it, which would be wrong. I'll add a comment too.

Reply:
Thanks for clarifying.